lib/solr_wrapper/instance.rb

```diff
@@ -41,9 +45,10 @@ def wrap(&_block)
     ##
     # Start Solr and wait for it to become available
     def start
-      extract
+      raise_error_unless_extracted
```

Comment:
I don't remember why I removed that.  I think it had to do with annoyances that I ended up fixing elsewhere.